exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -64,7 +68,7 @@ Options:
 Function name: `:datetime`
 
 Aliases:
-- (none currently; should there be `:date` and `:time` aliases to shorthand getting particularly timeStyle?)
+- `:date` and `:time` have been proposed and will be considered for 45
```

Comment:
I think we should be explicit here about whether these are in or out. Or is that intentionally left for later consideration? Also, as currently envisioned these aren't technically "aliases". Maybe "sub-functions"? I don't really care how they're categorised, mind.

```suggestion
- `:date` (with `style=...` option corresponding to `:datetime dateStyle=...`)
- `:time` (with `style=...` option corresponding to `:datetime timeStyle=...`)
```